src/message.ts

```diff
@@ -0,0 +1,93 @@
+import fs = require("fs");
+import q = require("q");
+import logger from "./logger";
+import pactUtil, {DEFAULT_ARG, SpawnArguments} from "./pact-util";
+import pactStandalone from "./pact-standalone";
+import path = require("path");
+const mkdirp = require("mkdirp");
+const checkTypes = require("check-types");
+
+export class Message {
+	public readonly options: MessageOptions;
+	private readonly __argMapping = {
+		"content": DEFAULT_ARG,
```

Comment:
I'm not sure that we can do 2 default args?  It's also a fairly bad CLI practice.

Reply:
It seems to work, at least, the integration test does.

Reply:
But why both?  Are they both mandatory?

Reply:
Yep, they're both mandatory.

Reply:
alright, but I'm not a fan :/